site/src/Blockquote/Blockquote.tsx

```diff
@@ -2,7 +2,11 @@ import { ReactNode } from 'react';
 import { Box } from '../system';
 import * as styles from './Blockquote.css';
 
-export default (props: { children: ReactNode }) => {
+export interface BlockquoteProps {
```

Comment:
This, any probably all the other `export`s you've added to prop types, are not necessary because those types aren't used outside the files they're defined in. They can be removed.